lib/httparty.rb

```diff
@@ -529,6 +532,12 @@ def options(path, options = {}, &block)
 
     private
 
+    def ensure_method_maintained_across_redirects(options)
+      unless options.has_key? :maintain_method_across_redirects
```

Comment:
Exactly. I wanted to make sure there was still a way to override the new behavior, in case someone wanted to do that.